include/install-vitasdk.sh

```diff
@@ -22,7 +26,11 @@ install_vitasdk () {
         sudo mkdir -p $INSTALLDIR
         sudo chown $USER:$(id -gn $USER) $INSTALLDIR
       fi
-      wget -O- "$(get_download_link master linux)" | tar xj -C $INSTALLDIR --strip-components=1
+      if ! [[ "$(uname -m)" =~ ^(armv7l|arm64|aarch64)$ ]]; then
+        wget -O- "$(get_download_link master linux)" | tar xj -C $INSTALLDIR --strip-components=1
```

Comment:
`get_download_link` script should support various platforms and also the results of `autobuilds` must have multiplatform files instead of using an external fork. (currently, this repo only has shipped x86_64 files)